stdlib/_socket.pyi

```diff
@@ -456,6 +497,10 @@ if sys.platform == "linux":
     RDS_RDMA_USE_ONCE: int
     RDS_RECVERR: int
 
+    # This is supported by cpython but doesn't seem to be a real thing.
```

Comment:
```suggestion
    # This is supported by CPython but doesn't seem to be a real thing.
```

Reply:
If you want extra brownie points, you could consider opening an issue at CPython about this ;)

Reply:
CPython issue for this constant: https://github.com/python/cpython/issues/127069